src/test/java/org/mule/service/http/impl/functional/server/AbstractHttpServerTestCase.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.service.http.impl.functional.server;
+
+import org.mule.runtime.http.api.server.HttpServer;
+import org.mule.runtime.http.api.server.HttpServerConfiguration;
+import org.mule.service.http.impl.functional.AbstractHttpServiceTestCase;
+import org.mule.tck.junit4.rule.DynamicPort;
+
+import org.junit.After;
+import org.junit.Rule;
+
+public abstract class AbstractHttpServerTestCase extends AbstractHttpServiceTestCase {
```

Comment:
Where is this used?

Reply:
in the EE part, or in whatever additional place we want to add a test. I actually stole this from the master branch, where it also exists here.

Reply:
Ah, yes, more master stuff, ok.